tests/coalaCITest.py

```diff
@@ -105,6 +105,9 @@ def test_coala_delete_orig(self):
              make_temp(suffix='.coafile', prefix='', dir=tempdir) as coafile,\
              make_temp(dir=tempdir) as unrelated_file:
             orig_file.close()
+            with open(coafile, mode="w") as file:
+                file.write("files = something\n")
+
```

Comment:
Why is this here ? Is it needed for `test_coala_delete_orig` ? That's confusing don't you think ?